<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventView</class>
 <widget class="QDialog" name="EventView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="hourLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kenyan Coffee</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hours:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hourInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Neou</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Event Name: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="nameInput">
      <property name="font">
       <font>
        <family>Neou</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Text: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="categoryLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Category: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="categoryInput">
      <property name="font">
       <font>
        <family>Neou</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="dateInput">
      <property name="font">
       <font>
        <family>Neou</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
